spaceship.zsh

```diff
@@ -446,11 +448,19 @@ spaceship_host() {
 spaceship_dir() {
   [[ $SPACESHIP_DIR_SHOW == false ]] && return
 
-  _prompt_section \
-    "$SPACESHIP_DIR_COLOR" \
-    "$SPACESHIP_DIR_PREFIX" \
-    "%${SPACESHIP_DIR_TRUNC}~" \
-    "$SPACESHIP_DIR_SUFFIX"
+  if _is_git && [[ $SPACESHIP_DIR_TRUNC_IN_GIT == true ]]; then
+    _prompt_section \
+      "$SPACESHIP_DIR_COLOR" \
+      "$SPACESHIP_DIR_PREFIX" \
+      "$(basename $(git rev-parse --show-toplevel))" \
```

Comment:
Does this show sub directories ? Now I'm getting only the root directory. See here, _logos_ sub directory is not shown.

![image](https://user-images.githubusercontent.com/10276208/28134713-6aab6898-6761-11e7-99b6-cc373154d9d9.png)